tests/test-sources/plugins/utils/scope.nix

```diff
@@ -0,0 +1,21 @@
+{
+  empty = {
+    plugins.scope.enable = true;
+  };
+
+  defaults = {
+    plugins.scope = {
+      enable = true;
+
+      settings = {
+        hooks = {
+          pre_tab_enter.__raw = ''
+            function()
+              print("About to enter tab")
+            end
+          '';
```

Comment:
If this really is the plugin-default, it should also be reflected in the option declaration.

Ideally, the `defaults` test should explicitly set the plugin-default for _all_ declared settings options.

If this is more of an "example" test-case, that's fine too. But it should be reflected in the test-case name.